### DISCUSSION TITLE
Showing Invalid Configuration when added costume domain

### DISCUSSION THREAD

**Discussion by DETAIT** [vercel/community#1574] [Help] [answered]
### Summary

hello
 
I have pointed the Cname to my service provider and it has propagated but it's showing invalid configuration.
any help will be appreciated.

### Additional information

![image](https://user-images.githubusercontent.com/122367630/220023608-eba7e604-19f5-4840-ba29-a9ea4c9f71bb.png)

**Comment by swarnava** [comment, marked as answer]
Hi,

Did you set the CNAME as instructed in the dashboard? I don't see it propagated yet: https://dnschecker.org/#CNAME/admin.faspro24.com

**Reply by DETAIT** [reply]
> Hi,
> 
> Did you set the CNAME as instructed in the dashboard? I don't see it propagated yet: https://dnschecker.org/#CNAME/admin.faspro24.com

Thank you

**Comment by jedahan** [comment]
Same issue with our config. https://dnschecker.org/#CNAME/beta.pluspool.org shows it has propagated.

 
![Screenshot 2023-03-08 at 12 32 04 PM](https://user-images.githubusercontent.com/32407/223787437-2911a210-8ebc-4d98-8274-739a2f31ff11.png)
![Screenshot 2023-03-08 at 12 32 17 PM](https://user-images.githubusercontent.com/32407/223787438-39ddd2b0-f163-4f89-90ae-76dd2feefe74.png)

**Comment by jedahan** [comment]
My wild guess - the error message shows if either CNAME or Nameservers are invalid - so for people who are only using one and not the other, its confusing.

If this is the case, maybe the error message should be scoped to the CNAME / Nameservers tabs.

**Reply by smaeda-ks** [reply]
@jedahan Your case is probably by a different cause. Have you run `dig +trace beta.pluspool.org`? What I see from this is your authoritative name servers (`ns1.mediatemple.net.` or `ns2.mediatemple.net.`) appear to be returning an invalid glue record, which causes the DNS query to fail (timed out error).

```
$ dig beta.pluspool.org @ns2.mediatemple.net. +noall +auth
beta.pluspool.org.	0	IN	NS	KLQRVamOiKeR.pluspool.org.

$ dig beta.pluspool.org @KLQRVamOiKeR.pluspool.org.
;; communications error to 68.178.223.225#53: timed out
;; communications error to 68.178.223.225#53: timed out
;; communications error to 68.178.223.225#53: timed out

; <<>> DiG 9.18.12 <<>> beta.pluspool.org @KLQRVamOiKeR.pluspool.org.
;; global options: +cmd
;; no servers could be reached
```
